src/create_helpers.ts

```diff
@@ -1,67 +1,77 @@
 
-function validateClassLikeChildren(children: ts.Node[] | undefined, validate: (n: ts.Node) => boolean, msg: string)
-{
+function validateClassLikeChildren(
+    children: ts.Node[] | undefined,
+    validate: (n: ts.Node) => boolean,
+    msg: string,
+) {
     // Validate that the children array actually contains type elements.
     // This should never trigger, but is here for safety.
-    if (children)
-    {
-        for (let i = children.length - 1; i >= 0; --i)
-        {
+    if (children) {
+        for (let i = children.length - 1; i >= 0; --i) {
             const child = children[i];
-            if (!validate(child))
-            {
-                warn(`Encountered child that is not a ${msg}, this is likely due to invalid JSDoc.`, child);
+            if (!validate(child)) {
+                warn(
+                    `Encountered child that is not a ${msg}, this is likely due to invalid JSDoc.`,
+                    child,
+                );
                 children.splice(i, 1);
             }
         }
     }
 }
 
-function validateClassChildren(children: ts.Node[] | undefined)
-{
-    return validateClassLikeChildren(children, ts.isClassElement, 'ClassElement');
+function validateClassChildren(children: ts.Node[] | undefined) {
+    return validateClassLikeChildren(
+        children,
+        ts.isClassElement,
+        'ClassElement',
+    );
 }
 
-function validateInterfaceChildren(children: ts.Node[] | undefined)
-{
+function validateInterfaceChildren(children: ts.Node[] | undefined) {
     return validateClassLikeChildren(children, ts.isTypeElement, 'TypeElement');
 }
 
-function validateModuleChildren(children?: ts.Node[])
-{
+function validateModuleChildren(children?: ts.Node[]) {
     // Validate that the children array actually contains declaration elements.
     // This should never trigger, but is here for safety.
-    if (children)
-    {
-        for (let i = children.length - 1; i >= 0; --i)
-        {
+    if (children) {
+        for (let i = children.length - 1; i >= 0; --i) {
             const child = children[i];
-            if (!ts.isClassDeclaration(child)
-                && !ts.isInterfaceDeclaration(child)
-                && !ts.isFunctionDeclaration(child)
-                && !ts.isEnumDeclaration(child)
-                && !ts.isModuleDeclaration(child)
-                && !ts.isTypeAliasDeclaration(child)
-                && !ts.isVariableStatement(child))
-            {
-                warn('Encountered child that is not a supported declaration, this is likely due to invalid JSDoc.', child);
+            if (
+                !ts.isClassDeclaration(child) &&
+                !ts.isInterfaceDeclaration(child) &&
+                !ts.isFunctionDeclaration(child) &&
+                !ts.isEnumDeclaration(child) &&
+                !ts.isModuleDeclaration(child) &&
+                !ts.isTypeAliasDeclaration(child) &&
+                !ts.isVariableStatement(child)
```

Comment:
Definitely prefer my previous style of having the first item be on the same line as if, and have the operator precede the statement.

Reply:
This isn't an option with prettier